test_fms/mpp_io/test_mpp_io.F90

```diff
@@ -39,9 +39,7 @@ program test
 
   implicit none
 
-#ifdef use_netCDF
 #include <netcdf.inc>
-#endif
```

Comment:
If this is the only place the macro exists in this file, I don't know that we even need the #include

Reply:
I'll make sure this line is/is not needed, and delete if not. Thank you.

Reply:
Not needed.  New version pushed.